packages/webservice/app/build.gradle.kts

```diff
@@ -3,6 +3,15 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 description = "app"
 
+val environment: String by project
+var build_env_final : String
+
+if (!project.hasProperty("environment")){
+  build_env_final = "development"
+}else{
+  build_env_final = environment
+}
```

Comment:
does this work?

```suggestion
val buildEnv = environment ?: "development"
```

Reply:
this doesn't work as expected. i can convert the if condition into a single line to be as terse as your proposed.